Lens/Grammar/Grammar.v2.txt

```diff
@@ -156,15 +156,15 @@ bool <BoolNode>                             = "true" | "false"
 
 // specials
 unary_op                                    = "-" | "not"
-binary_op                                   = "|" | "&" | "^" | "&&" | "||" | "^^" | "<:" | ":>" | "==" | "<>" | ">" | "<" | "<=" | ">=" | "+" | "-" | "*" | "/" | "%" | "**"
+binary_op                                   = "|" | "&" | "^" | "&&" | "||" | "^^" | "<:" | ":>" | "==" | "<>" | ">" | "<" | "<=" | ">=" | "+" | "-" | "*" | "/" | "%" | "**" | "??"
```

Comment:
Whoops. This is for an upcoming feature which slithered its way into this pull request.